mailbox/store/src/main/java/org/apache/james/mailbox/store/SimpleMailboxMetaData.java

```diff
@@ -45,7 +45,6 @@ public SimpleMailboxMetaData(MailboxPath path, MailboxId mailboxId, char delimit
     }
 
     public SimpleMailboxMetaData(MailboxPath path, MailboxId mailboxId, char delimiter, Children inferiors, Selectability selectability) {
-        super();
```

Comment:
why? + unrelated to this commit. Really, please try to not mix big commits with (even little) changes